django-stubs/middleware/csrf.pyi

```diff
@@ -24,7 +24,11 @@ def rotate_token(request: HttpRequest) -> None: ...
 class CsrfViewMiddleware(MiddlewareMixin):
     def process_request(self, request: HttpRequest) -> None: ...
     def process_view(
-        self, request: HttpRequest, callback: Callable | None, callback_args: tuple, callback_kwargs: dict[str, Any]
+        self,
+        request: HttpRequest,
+        callback: Callable[..., Any] | None,
```

Comment:
My goal was to match the existing type unless there was an obvious improvement. For example, when replacing `Callable` I'd use `reveal_type()` to ensure I'm matching it 1:1. I tried very hard to avoid adding `Any` unless that's truly what was previously being inferred.